### ISSUE TITLE
ADDONS:TR:AVAILABLE returns false even when TR addon is installed

### ISSUE THREAD

**Issue by ezeeetm** [KSP-KOS/KOS#2303]
```
if ADDONS:TR:AVAILABLE {
    PRINT "Trajectories is available.".
} else {
    PRINT "Trajectories is not available.".
}
```

Returns `Trajectories is not available`

But I have it installed (below).  **Are we required to lock to a certain version of Trajectories, and if so which one?**

Thanks!

![image](https://user-images.githubusercontent.com/9197727/43049503-e041519e-8dc6-11e8-8dca-b8f60a7b5b7f.png)

**Comment by PiezPiedPy** [comment]
Changes where made to the Trajectories API which makes versions of Trajectories from v2 onwards incompatible with the current release of kOS
You can use my development build of kOS that has the necessary changes https://www.dropbox.com/s/ykvqsqlgc8714h3/kOS_develop.zip?dl=0

**Comment by ezeeetm** [comment]
@PiezPiedPy ty.  super helpful!

**Comment by ezeeetm** [comment]
confirmed working w Trajectories v2.2.1

![image](https://user-images.githubusercontent.com/9197727/43050656-ec6479e2-8dda-11e8-9d54-2a65d77a060c.png)

**Comment by PiezPiedPy** [comment]
@ezeeetm FYI There are new functions available: Re #2220

Changes for v2.0.0 are:
* Changes to Trajectories addon classes to support the v2.0.0 release of Trajectories.
* New API function added `HasTarget`.

Changes for v2.2.0 are:
* Changes to Trajectories addon classes to support the v2.2.0 release of Trajectories.
* New API functions added `TimeTillImpact`.
* For the Descent Profile `Prograde` and `Retrograde`.
* And some for version checking `GetVersion` `IsVerTwo` `IsVerTwoTwo`.

Also note:
 `SetTarget` can be used regardless of a predicted trajectory calculated or not and thus a check for `HasImpact` is not required but an active vessel check is still required.
`PlannedVector` and `CorrectedVector` require a valid target to be set before calling.

**Comment by PiezPiedPy** [comment]
Also see the https://github.com/KSP-KOS/KOS/blob/develop/doc/source/addons/Trajectories.rst
